.github/workflows/presubmit.yaml

```diff
@@ -1,10 +1,24 @@
-name: CI
+name: Presubmit 
 
 on: [pull_request, push]
 
 jobs:
-  build:
+  build-orchestrator:
     runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@a81bbbf8298c0fa03ea29cdc473d45769f953675 # aka v2
+    - name: Install dependencies
+      uses: actions/setup-go@v3
+      with:
+        go-version: '1.13.15'
+    - run: go version
+    - name: Vet Test Build
+      run: cd frontend/src/host_orchestrator && go vet ./... && go test ./... && go build ./...
```

Comment:
nit: `go doc cmd/vet` says that the `./...` is not required, it uses CWD by default.

It also says that not everything it detects are genuine problems, that could be an issue when added to presubmit checks.